docs/source/examples/spot-jobs.rst

```diff
@@ -6,57 +6,89 @@ Managed Spot Jobs
 SkyPilot supports managed spot jobs that can **automatically recover from preemptions**.
 This feature **saves significant cost** (e.g., up to 70\% for GPU VMs) by making preemptible spot instances practical for long-running jobs.
 
-To maximize availability, SkyPilot automatically finds available spot resources across regions and clouds.
-Here is an example of BERT training job failing over different regions across AWS and GCP.
+SkyPilot automatically finds available spot resources across regions and clouds to maximize availability.
+Here is an example of a BERT training job failing over different regions across AWS and GCP.
 
 .. image:: ../images/spot-training.png
   :width: 600
   :alt: BERT training on Spot V100
 
-Below are requirements for using managed spot jobs:
+Below are the requirements for using managed spot jobs:
 
-(1) **Mounting code and datasets**: Local file mounts/workdir are not supported. Cloud buckets should be used to hold code and datasets, which can be satisfied by using :ref:`SkyPilot Storage <sky-storage>`.
+(1) **Task YAML**: Managed Spot requires a YAML to describe the job, tested with :code:`sky launch`.
+(2) **Checkpointing and recovery**(optional): For job recovery with less progress loss, application code can checkpoint periodically to a
```

Comment:
Need a space before (optional)? for the bold font to correctly show

Reply:
Is it better?: For job recovery with less progress loss -> For job recovery with progress resuming

Reply:
Updated. Thanks!